package/test-package.sh

```diff
@@ -16,22 +16,29 @@ kevm kast tests/interactive/log3_MaxTopic_d0g0v0.json kast  --backend llvm > tes
 git --no-pager diff --no-index --ignore-all-space -R tests/interactive/log3_MaxTopic_d0g0v0.json.parse-out tests/interactive/log3_MaxTopic_d0g0v0.json.parse-expected
 rm -rf tests/interactive/log3_MaxTopic_d0g0v0.json.parse-out
 
-kevm run tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json --backend llvm \
-    --mode NORMAL --schedule BERLIN --chainid 1                                               \
-    > tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json.llvm-out          \
-    || git --no-pager diff --no-index --ignore-all-space -R tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json.llvm-out tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json.expected
-rm -rf tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json.llvm-out
+# This test currently segfaults on M1 Macs
+if [ "${APPLE_SILICON:-false}" == "false" ]; then
```

Comment:
```suggestion
if ${APPLE_SILICON:-false}; then
```

`false` is a command on most systems that can be used directly for conditionals. Same with `true`.